examples/main.py

```diff
@@ -31,20 +29,18 @@
 with dai.Pipeline() as pipeline:
     cam = pipeline.create(dai.node.Camera).build()
 
+    image_type = dai.ImgFrame.Type.BGR888p
+    if "gray" in model:
+        image_type = dai.ImgFrame.Type.GRAY8
+
     # YOLO and MobileNet-SSD have native parsers in DAI - no need to create a separate parser
     if parser_name == "YOLO" or parser_name == "SSD":
-        network = pipeline.create(dai.node.DetectionNetwork).build(
-            cam.requestOutput(
-                input_shape, type=dai.ImgFrame.Type.BGR888p, fps=fps_limit
-            ),
+        nn = pipeline.create(dai.node.DetectionNetwork).build(
```

Comment:
Note: ParsingNeuralNetwork can now be also used for models with "YOLO and "SSD" parsers so no need for separate if here anymore. But this can be part of separate PR.